Source/VFECore/Genes/Harmony/HumanlikeMeshPoolUtility_Patches.cs

```diff
@@ -0,0 +1,345 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection.Emit;
+using System.Reflection;
+using System.Linq;
+using HarmonyLib;
+using UnityEngine;
+using VFECore;
+using RimWorld;
+using Verse;
+
+
+namespace VanillaGenesExpanded
```

Comment:
Isn’t want to switch the namespace to one for VEF, so it doesn’t get confusing which mod adds this.